doc/source/building/windows.rst

```diff
@@ -168,90 +281,115 @@ the following in your build log:**
 
    FOUND:
       libraries = ['openblas']
-      library_dirs = ['C:\opt\64\lib']
-      language = c
+      library_dirs = ['C:\...........\lib']
+      language = f77
       define_macros = [('HAVE_CBLAS', None)]
 
+We'll get back to this below how to check.
+
 Building SciPy
 ==============
 
-Once you have built OpenBLAS, it's time to build SciPy. Before continuing make sure to
-install the following software for building on the latest Python version. For building
-on other Python versions, see the WindowsCompilers_ page.
+Once you have built OpenBLAS, it's time to build SciPy. Before continuing make
+sure to install the following software for building on the latest Python
+version. For building on other Python versions, see the WindowsCompilers_ page.
+We are also assuming that your Python is on the system path. That is to say,
```

Comment:
Note that Python 3,8 was installed in MinGW, and that is on the MinGW path (which is different from the system path). Just to confirm - we care about what is on the _system_ path, not whatever path MinGW uses?

Reply:
After the OpenBLAS built. We won't have anything else to do with MSYS hence this section shouldn't confuse. I've made some remarks about this but let me add more.

Reply:
No I think it's fine. I see that you made some remarks about it now.